.github/workflows/lint-and-tests.yml

```diff
@@ -63,7 +63,11 @@ jobs:
       - name: Run e2e tests
         shell: bash
         run: scripts/run.e2e.sh
+      - name: Check Snapshots
+        if: always()
+        run: ls /home/runner/.avalanche-cli/snapshots
       - name: 'Upload Artifact'
+        if: always()
```

Comment:
why this if?

Reply:
It makes sure that artifacts still upload if the previous command fails. Otherwise, this step will be skipped and we will only get logs if the tests pass.